com.unity.netcode.gameobjects/README.md

```diff
@@ -1,3 +1,9 @@
+**For bug reports, please follow the standard [Unity Bug Reporting](https://unity3d.com/unity/qa/bug-reporting) process.**
```

Comment:
I think this notice line is optional because this README.md file is inside the package while the other root one is not inside the package. When people land into this repository home page, they will see the other notice message anyway.

I put this here to discuss about it but I actually want to remove this line from the package's README.md file.
(Because people using UnityEditor will probably use Help > Report a Bug and only people will use GitHub for bug reports will come over GitHub web interface which in that case will see the root README.md's message anyway)

What are your thoughts?

Reply:
Agree, we don't need this in the package's readme. Only need it in the root readme.